driver-core/src/main/com/mongodb/client/model/expressions/ArrayExpression.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.expressions;
+
+public interface ArrayExpression<T extends Expression> extends Expression {
```

Comment:
`./gradlew clean check jar testClasses docs -x test -x integrationTest` emits multiple warnings about doc comments missing on the public API program elements added in this PR (there are no such warnings for the existing code). We need to add doc comments to get rid of the warnings.

Reply:
+1 and I don't know why Checkstyle doesn't catch a lot of these.